src/db/spanner/models.rs

```diff
@@ -1352,10 +1552,12 @@ impl SpannerDb {
                 let (columns, values) = bso_to_update_row(&user_id, collection_id, bso, timestamp)?;
                 load_size += values.compute_size() as usize;
                 updates.entry(columns).or_insert_with(Vec::new).push(values);
+                self.metrics.incr("storage.spanner.update");
```

Comment:
I don't think a counter for updates/inserts will tell us much -- I think what we want is a histogram metric with the whole counts, e.g. this request contained x inserts, and x updates.

These stats are much more important within batch update (plain POSTs are capped at 100 bsos). The counts there would be the results of the update/insert calls. The question we wanted answered with this was: on average, how much do bulk writes (batches) insert vs update